.azure-pipelines/release.yml

```diff
@@ -0,0 +1,85 @@
+trigger:
+  tags:
+    include:
+      - '*.*.*'
+
+pr: none
+
+stages:
+
+  - stage: 'build_conda_package'
+    displayName: 'Build Conda Package'
+    jobs:
+      - job: 'linux'
+        displayName: 'Linux'
+        timeoutInMinutes: 120
+        strategy:
+          matrix:
+            Python37:
+              PYTHON_VERSION: '3.7'
+        pool:
+          vmImage: 'ubuntu-latest'
+        variables:
+          - group: tokens
+        steps:
+          - checkout: self
+            submodules: true
+          - bash: | 
+              echo "##vso[task.prependpath]$CONDA/bin"
+              echo "##vso[task.setvariable variable=conda_dir]$CONDA"
+            displayName: 'add conda to PATH'
+          - bash: |
+              set -ex
+              conda --version
+              conda install --yes anaconda-client conda-build
+              conda config --set always_yes yes --set changeps1 no
+            displayName: 'Conda configuration'
+          - bash: |
+              conda build --user scipp --token "$ANACONDA_TOKEN" --channel conda-forge --label dev --python="$PYTHON_VERSION" .
+            env:
+              ANACONDA_TOKEN: $(anaconda_token_secret)
+            displayName: 'Package build'
+          - task: PublishBuildArtifacts@1
+            inputs:
+              PathtoPublish: "$(conda_dir)/conda-bld/linux-64"
+              ArtifactName: 'linux-64'
+            displayName: 'Publish Conda package artefacts'
+      - job: 'mac'
+        displayName: 'Mac OS'
+        timeoutInMinutes: 120
+        strategy:
+          matrix:
+            Python37:
+              PYTHON_VERSION: '3.7'
+        pool:
+          vmImage: 'macOS-10.14'
```

Comment:
There seems to be a mismatch between 10.14 here and 10.15 4 lines below. Not sure how crucial that is

Reply:
That's the difference between the OS version and the SDK version see https://en.wikipedia.org/wiki/Xcode